README.md

```diff
@@ -16,6 +16,8 @@ To create the schema, set the environment variable `SOLR_HOST` to the appropriat
 ```bash
 export SOLR_HOST=192.168.99.100:32080
 
+After doing this you will need to copy `scatlas.owl` file to your running solrcloud containers. The change you need to make is in `create-scxa-analytics-schema.sh`. Set this path `SCXA_ONTOLOGY=${SCXA_ONTOLOGY:-"file:///srv/gxa/scatlas.owl"}` to appropriate path in container where you copied the `scatlas.owl` file. You can download this file from one of the vms running our solr instances. Here you can find the name of the VMs: https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=GXA&title=Atlas+development+environments
```

Comment:
Probably the only thing that you need to say here is:
>Set the SCXA_ONTOLOGY environment variable to the path of the owl file as mounted inside the container.

no need to mention create-scxa-analytics-schema.sh

If you are after having a more detailed set of instructions for setting up the developers test environment (which is one use case of this repo), I would suggest that the best place for that is an SOP where you can include more details, internal URLs, etc.

Reply:
This might come in handy: https://www.ebi.ac.uk/seqdb/confluence/display/GXA/GEGDEVSOP005%3A+Deploy+SolrCloud+on+Kubernetes+for+Single+Cell+Expression+Atlas+continuous+integration#GEGDEVSOP005:DeploySolrCloudonKubernetesforSingleCellExpressionAtlascontinuousintegration-3.5.BioSolrontology